settings/Civixero.setting.php

```diff
@@ -104,4 +104,23 @@
     ),
     'quick_form_type' => 'Element',
   ),
+  'xero_push_contribution_status' => array(
+    'group_name' => 'Xero Settings',
+    'group' => 'xero',
+    'name' => 'xero_push_contribution_status',
+    'type' => 'Array',
+    'add' => '4.7',
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'description' => 'Select contribution status that can be pushed to Xero.',
+    'title' =>  'Push Contribution Status',
+    'help_text' => '',
+    'html_type' => 'Select',
+    'options' => CRM_Contribute_PseudoConstant::contributionStatus(NULL, 'name'),
```

Comment:
I think this syntax is 'more correct' https://github.com/litespeedmarc/civicrm-core/blob/dfc3d2e798940e3837e842e41e3d16a009760764/settings/Core.setting.php#L46-L46